sdk/core/azure-core/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Breaking Changes
 
 - Removed the `AzureNoReturnPath()` function from the global namespace, and deprecated the associated macros, such as `AZURE_ASSERT` since they are meant for internal use only. If your code was using the `AZURE_ASSERT` macro, consider using the standard library's `assert` as an alternative.
+- The `Base64::Decode` API will throw a `std::runtime_error` exception if presented with invalid inputs.
```

Comment:
I wonder if we should consider this as a bug fix rather than breaking change, since it isn't source breaking, rather just a behavioral breaking change that most customers wouldn't even rely on.

Reply:
We have an existing test which broke with this change (in an earlier version) so I think calling it out makes sense. However I could see calling it out as a bug fix instead of breaking change (changing the section for the changelog).